prefect_gcp/bigquery.py

```diff
@@ -177,18 +178,16 @@ async def bigquery_create_table(
             of the newly created table
         project: Project to initialize the BigQuery Client with; if
             not provided, will default to the one inferred from your credentials.
-        location: location of the dataset that will be written to.
-
+        location: The location of the dataset that will be written to.
+        external_config: The [external data source](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_table#nested_external_data_configuration).
```

Comment:
One last suggestion and it's all good to go!
```suggestion
        external_config: The [external data source](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_table#nested_external_data_configuration).  # noqa
```